epicmaxco/vuestic-ui
# Replace jest with vitest

I tried using vitest, and it went pretty fine.

Vitest is a good solution, as it integrates with vite nicely, so we don't have to use separate environment for testing. It's also very fast.

What's done:
* replace jest with vitest
* use vitest on precommit
* ensure tests are green or disabled
* fixing disabled tests moved to this issue - https://github.com/epicmaxco/vuestic-ui/issues/2133
